clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -128,6 +160,15 @@ public abstract CreateTopicsResult createTopics(Collection<NewTopic> newTopics,
      *
      * This operation is supported by brokers with version 0.10.1.0 or higher.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link DeleteTopicsResult#values() values()} method of the  returned {@code DeleteTopicsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.UnknownTopicOrPartitionException}
+     *     if the topic does not exist</li>
+     *     <li>{@link org.apache.kafka.common.errors.NotControllerException}
+     *     if this is not the correct controller for the cluster</li>
```

Comment:
This exception is only thrown if the controller for the cluster cannot be found.